source/Random.h

```diff
@@ -39,8 +40,13 @@ class Random {
 	static uint32_t Polya(uint32_t k, double p = .5);
 	// Get a number from a binomial distribution (i.e. integer bell curve).
 	static uint32_t Binomial(uint32_t t, double p = .5);
-	// Get a normally distributed number (mean = 0, sigma= 1).
-	static double Normal();
+	// Get a normally distributed number (mean = 0, sigma = 1) using std::normal_distribution.
+	static double StdNormal();
```

Comment:
This function is not needed if you provide default arguments to the function below.

Reply:
I left the no args version so I wouldn't disturb other code needlessly.  This is related to [this comment](https://github.com/endless-sky/endless-sky/pull/6803#discussion_r875776947) in that I'll be glad to refactor the usages of the old Normal functions as long as someone approves it.  Not trying to put up barriers for mergeability for myself.

Reply:
I think I understand what you mean now...that `int x = 0` stuff.  Yeah that's a good point.  Let me see if I can get it working.

Reply:
Fixed in 70c45ba15190fb2fdda3ebaa689cf2c09648d6e7.